chapters/classes.tex

```diff
@@ -1478,7 +1478,7 @@ \subsection{Real Type}\label{real-type}
   parameter StringType quantity    = "";
   parameter StringType unit        = "" "Unit used in equations";
   parameter StringType displayUnit = "" "Default display unit";
-  parameter RealType min = -Inf, max = +Inf; // Inf denotes a large value
+  parameter RealType min = ..., max = ...; //  See below
```

Comment:
Formatting:
```suggestion
  parameter RealType min = $\ldots$, max = $\ldots$; //  See below
```

Reply:
But why have anything at all, considering that no default is given for `start`?
```suggestion
  parameter RealType min; // Minimum allowed value.
  parameter RealType max; // Maximum allowed value.
```

Reply:
By the way, why do some of the attributes have explanations in comments, while some (`unit` and `displayUnit`) have description strings?

Reply:
> But why have anything at all, considering that no default is given for `start`?

I believe I tried that earlier, and @beutlich complained about that variant.
Since we have the assert using `min` I can see that it might require more.

As for `unit `and `displayUnit `having description strings and some attributes having comments: it has been that way for some time, I agree that it should likely be cleaned up at some point - it's just that we need reviews for existing topics before opening additional ones.

(Oh, and `start` not having any value was a very deliberate choice.)